port        : /dev/ttyUSB0
baudrate    : 115200
write_time  : 0.012